docs/freefeatures.rst

```diff
@@ -306,25 +310,26 @@ will continue to the end::
         3  alias create three !echo three
         4  alias create four !echo four
 
-You can use negative numbers as either the first or second number of the range (but not both). If
-you want to display the last three commands entered::
+You can use negative numbers as either the first or second number of the range
+(but not both). If you want to display the last three commands entered::
```

Comment:
This is a weird limitation.  But if is well documented so no complaints.

Reply:
Fixed this so it works if both indices are negative.